interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -14,7 +14,16 @@ angular.module('adage.analyze.sampleBin', [
 
 .factory('SignatureSet', ['$resource', 'ApiBasePath',
```

Comment:
@mhuyck: I have been wondering whether it is a good idea to move all signature-related resource classes to `signature/resources.js` file but haven't found time to do it yet. Since you are working on this service, what do you think?

After this service is moved to `signature/resources.js`, you can also remove the `ngResource` dependency on line 8.